docs/draft/mentions.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Mentions
+nav_order: 2
+has_children: true
+permalink: /specs/draft
```

Comment:
Number

Also, is this a draft or is it stable? I.e. is it describing what is live already?